content/module-reference/processing-modules/negadoctor.md

```diff
@@ -64,17 +72,22 @@ The settings on this tab should not be needed for most well-preserved negatives
 The other case where these color cast corrections may be needed is if the white balance properties of the light used to scan the film negative are significantly different to the light source under which the original film camera took the shot. For example, if you illuminate the film with an LED light, but the original shot was taken under daylight, this may require some additional color cast corrections.
 
 shadows color cast
-: These three sliders allow you to correct for color casts in the shadows by adjusting the _red_, _green_ and _blue_ channels individually. Use the color picker to set the sliders automatically by selecting a shadow region requiring correction. Because the shadows sliders can also affect highlights color casts, it is import to finish setting the shadows sliders before moving on to the highlights sliders.
+: These three sliders allow you to correct for color casts in the shadows by adjusting the _red_, _green_ and _blue_ channels individually. Use the color picker to set the sliders automatically by selecting a neutral grey shadow region requiring correction. Because the shadows sliders can also affect highlights color casts, it is import to finish setting the shadows sliders before moving on to the highlights sliders.
 
 highlights white balance
-: These three sliders allow you to correct the white balance in the highlights by adjusting the _red_, _green_ and _blue_ channels individually. Use the color picker to set the sliders automatically by selecting a region in the highlights that is not properly balanced.
+: These three sliders allow you to correct the white balance in the highlights by adjusting the _red_, _green_ and _blue_ channels individually. Use the color picker to set the sliders automatically by selecting a neutral grey region in the highlights that is not properly balanced.
 
 ## print properties
 
-This tab contains settings that will mimic the tonal effect of the photochemical papers that would have been used to create the hard copy image if you were not developing the photo digitally.
+This tab contains settings that will mimic the tonal effect of the photochemical papers that would have been used to create the hard copy image if you were not developing the photo digitally. These are creative settings, and serve a similar overall purpose to the creative tone curve settings on the _look_ tab of the [_filmic rgb_](filmic-rgb.md) module.
+
+It can be noted that the _print exposure_, _paper black_, _paper grade_ and _print exposure_ are analogous to the _slope_, _offset_ and _power_ controls in the [_color balance_](color-balance.md) module (which in turn is based on the ACES CDL standard). They bake a creative tone curve to enforce your contrast intent after the inversion, at the end of the module. The equation governing this slope/offset/power behaviour is:
```

Comment:
the color balance is based on the ASC (American society of cinematographers) CDL (color decision list). ACES is something else.

Reply:
Thanks @aurelienpierre. I'll correct it.

Reply:
done